WORKSPACE

```diff
@@ -0,0 +1 @@
+workspace(name = "rules_swiftnav")
```

Comment:
Question: What's the difference between this and the `WORKSPACE.bazel` file, which contain the same line?

Reply:
It depends on the version we are using. This was added because I was experimenting with a different bazel version. I will remove. Thanks for catching this.